infrastructure/helm-chart/values.yaml

```diff
@@ -1 +1,20 @@
-
+global:
+  containerRegistry: ghcr.io/airyhq
+security:
+  systemToken: ""
+  allowedOrigins: "*"
+config:
+  kafka:
+    brokers: "kafka-headless:9092"
+    zookeepers: "zookeeper:2181"
+    schemaRegistryUrl: "http://schema-registry:8081"
+    commitInterval: 1000
+provisioning:
+  kafka:
+    image: ghcr.io/airyhq/infrastructure/kafka
+    imageTag: 2.7.0
+tools:
+  segment:
+    enabled: true
+serviceAccount: airy-admin
+apiHost: http://airy.core
```

Comment:
We are generating some of the defaults with the `cli` now, but if we want for people to be able to install Airy with helm, without using the cli, I think the defaults should be in the chart.
Otherwise we should instruct people to do `helm create --init-only` (with some provider) and then run `helm install --values ./airy.yaml`.